Makefile

```diff
@@ -1,30 +1,30 @@
 .PHONY: check-style clean pre-run test install
 
 
-.npminstall: package.json
-	@if ! [ $(shell which npm) ]; then \
-		echo "npm is not installed"; \
+.yarninstall: package.json
+	@if ! [ $(shell which yarn) ]; then \
+		echo "yarn is not installed"; \
 		exit 1; \
 	fi
 
-	@echo Getting dependencies using npm
+	@echo Getting dependencies using yarn
 
-	npm install --ignore-scripts --no-package-lock
+	yarn install --ignore-scripts --no-package-lock
```

Comment:
does yarn has a `--ignore-scripts` flag?

do we really need to set `--no-package-lock` ?

Reply:
Yes, and I don't know.

Reply:
can you please test it and then report back?

Reply:
Yes sorry, I tested look over the help too fast and my mind made the match --no-lockfile with --no-package-lock automatically :(

`--pure-lockfile` sounds good to me